live-examples/html-examples/embedded-content/source.html

```diff
@@ -0,0 +1,12 @@
+<video controls
+    width="300"
+    height="200"
+    autoplay="true"
+    loop="true"
+    muted="true">
+    <source src="/media/examples/stream_of_water.webm" type="video/webm">
+    <source src="/media/examples/stream_of_water.mp4" type="video/mp4">
+    This browser does not support HTML5 video element.
```

Comment:
I think this should be:

"This browser does not support the HTML5 video element."